- Added Crop notifier when a player tries to place a crop too close t…

beerik94 opened the pull request:
This adds an onscreen message notification that the player is trying to place a crop too close to another crop.

radmint commented on valheim_plus.cfg:
```diff
@@ -248,6 +248,9 @@ disableCameraShake=false
 baseUnarmedDamage=10
 
+; When changed to true, makes you unable to place crops too close to eachother
+cropNotifier=false
```
I would put a bit more clear wording on the description. Maybe: "When changed to true, you will not be permitted to place a crop that is considered to close to another crop"

radmint reviewed (changes requested):
Thank you for the contribution!

1. Would you mind updating your summary to include what test cases you performed with this?
2. Did you try this feature in conjunction with the "no invalid placement" V+ mod under the [Building] section?
3. Please add a comment in your code addition to describe what it's doing

beerik94 commented:
Added comments and changed the config file.
This works even when you have "No invalid placement" turned ON.
I don't know if you want to keep it like that.
If not then I will have to look into fixing it.

radmint reviewed (approved):
These updates look great thank you @beerik94! Going to get a second set of eyes and test locally but then it should be ready to merge!

nxPublic commented:
Awesome work and solid implementation!

Thank you! ❤️
